tests/DateComparisonTest.php

```diff
@@ -11,7 +11,7 @@ final class DateComparisonTest extends TestCase
 {
     protected function setUp(): void
     {
-        CarbonImmutable::setTestNow(CarbonImmutable::create(2021, 5, 4));
+        CalendarSummaryTester::setTestNow(CarbonImmutable::create(2021, 5, 4));
```

Comment:
The new method `CalendarSummaryTester::setTestNow` has several parameters for year, month and day. But there it seems to be called with the wrong type?